common/src/main/webapp/less/style.less

```diff
@@ -1506,6 +1506,12 @@ html {
   color: var(--site-text-color);
 }
 
+.explore-help-icon-floating {
```

Comment:
This seems specific to the plot. Maybe the name should reflect that?

Reply:
Well I think there maybe more floating help icons in the future

Reply:
Agreed. But with the same positioning?